frontend/cypress/e2e/exercise.cy.ts

```diff
@@ -0,0 +1,897 @@
+            .its('hospitals')
+            .should('not.be.empty');
+
+        cy.log('rename a hospital')
+            .get('[data-cy="hospitalRenameInput"]')
+            .first()
+            .clear()
+            .type('ABC123');
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should('have.property', 'type', '[Hospital] Rename hospital');
+
+        cy.getState()
+            .its('exerciseState')
+            .its('hospitals')
+            .itsValues()
+            .firstElement()
+            .should('have.property', 'name', 'ABC123');
+
+        cy.log('update a hospitals transport time')
+            .get('[data-cy="hospitalUpdateTransportTimeInput"]')
+            .first()
+            .clear()
+            .type('30');
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[Hospital] Edit transportDuration to hospital'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('hospitals')
+            .itsValues()
+            .firstElement()
+            .should('have.property', 'transportDuration', 1800000);
+
+        cy.get('[data-cy=hospitalClosePopupButton]').click({ force: true });
+
+        cy.dragToMap('[data-cy=draggableTransferPointDiv]');
+        cy.get('[data-cy=openLayersContainer]').click();
+        cy.get('[data-cy=transferPointPopupHospitalNav]').click();
+        cy.get('[data-cy=transferPointPopupAddHospitalButton]').click();
+        cy.get('[data-cy=transferPointPopupAddHospitalDropdownButton]')
+            .first()
+            .click({ force: true });
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[TransferPoint] Connect hospital'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('transferPoints')
+            .itsValues()
+            .firstElement()
+            .its('reachableHospitals')
+            .should('not.be.empty');
+
+        cy.get('[data-cy=transferPointPopupCloseButton]').click();
+        cy.dragToMap('[data-cy=draggableVehicleDiv]');
+        cy.dragToMap('[data-cy=draggablePatientDiv]');
+
+        cy.get('[data-cy=chooseTransferTargetPopupHospitalDropdown]')
+            .first()
+            .click();
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[Hospital] Transport patient to hospital'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('hospitals')
+            .itsValues()
+            .firstElement()
+            .its('patientIds')
+            .should('not.be.empty');
+
+        cy.wait(150); // Do not trigger a double click
```

Comment:
On my machine it worked to just double click explicitly... that would get rid of the wait

Reply:
My problem was that clicking again would trigger a double click. You are saying that if you trigger a double click there, then it works?

Reply:
that would then be a double click to end the drop action and a single correct click

Reply:
You can try it by trippleclicking on an object. it zoomes in and opens the popup. It zoomes in which is not wanted but not bad either and imo it looks cleaner to have another click than a wait

Reply:
Fixed in b501980e368668c216da57909d2ea74c919bebe8

Reply:
Edit: Fixed in 483896ce042712d2fa69f3896bca3e1e5fd2d7ec